server/Makefile

```diff
@@ -332,11 +332,11 @@ app-layers: ## Extract interface from App struct
 	$(GO) run ./channels/app/layer_generators -in ./channels/app/app_iface.go -out ./channels/app/opentracing/opentracing_layer.go -template ./channels/app/layer_generators/opentracing_layer.go.tmpl
 
 i18n-extract: ## Extract strings for translation from the source code
-	$(GO) install github.com/mattermost/mattermost-utilities/mmgotool@mono-repo
+	$(GO) build -C ../tools/mmgotool -o $(GOBIN)/mmgotool .
```

Comment:
Not sure what the `-C` flag does. Can you clarify?

Reply:
It's just changing the directory for the `build` command. From the doc:
```
        -C dir
                Change to dir before running the command.
                Any files named on the command line are interpreted after
                changing directories.
```

Reply:
Ah okay, this is a new flag introduced from 1.20 onwards. We don't use 1.20 yet. Either I'd suggest to `cd`, or just use `go run`. 0/5